### ISSUE TITLE
No dialog box asking to update, when using "Sync mode"

### ISSUE THREAD

**Issue by sulliwane** [Microsoft/react-native-code-push#58]
After adding code-push to my project:
```
var CodePush = require("react-native-code-push")
CodePush.sync();
```
I see that my iphone is downloading the new bundle (after I release to code-push server). But no dialog box show up the alert about this new update, and it is automatically applied on the next start of the app.

Isn't the dialog box a default part of the "Sync mode" ?

**Comment by lostintangent** [comment]
We actually just changed the API to be "silent" by default, and to apply to update on the next restart instead of immediately.

If you want the dialog box, and immediate restart, the following is how you would configure it: sync({ updateDialog: true, installMode: CodePush.InstallMode.IMMEDIATE })

Apologies for the confusion. We updated the docs, but I just realized that the getting started section still shows he screenshot with the dialog. I'll update that.

Let us know if you have feedback about these defaults. We made this change based on user feedback.
